ssl/t1_lib.c

```diff
@@ -1416,10 +1392,41 @@ unsigned char *ssl_add_clienthello_tlsext(SSL *s, unsigned char *buf,
     s2n(0, ret);
 
     /*
+     * WebSphere application server can not handle having the
+     * last extension be 0-length (e.g. EMS, EtM), so keep those
+     * before SigAlgs
+     */
+    if (SSL_CLIENT_USE_SIGALGS(s)) {
```

Comment:
If we're not, then we don't send this extension and anyone who disables TLS 1.2 but enables either EtM or ExtMS will have trouble talking to the buggy origins.
But ... maybe we don't care.  The buggy origins are fairly rare, and should support TLS 1.2 fine as far as we know.